serverless/networking/serverless-ossm.adoc

```diff
@@ -10,6 +10,16 @@ toc::[]
 Using {ProductShortName} with {ServerlessProductName} enables developers to configure additional networking and routing options that are not supported when using {ServerlessProductName} with the default Kourier implementation.
 These options include setting custom domains, using TLS certificates, and using JSON Web Token authentication.
 
+[IMPORTANT]
+====
+Adding sidecar injection to pods in system namespaces such as `knative-serving` and `knative-serving-ingress` is not supported.
+====
+
+[IMPORTANT]
```

Comment:
Might be unavoidable, but normally we try to avoid back-to-back admonitions like this (especially of the same type).